components/ImageViewer.js

```diff
@@ -0,0 +1,178 @@
+import React, { Component } from 'react';
+import Icon from '@material-ui/core/Icon';
+import Zoom from '@material-ui/core/Zoom';
+import { withStyles } from '@material-ui/core/styles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const styles = theme => ({
+  root: {
+    height: "100%",
+    display: "flex",
+    cursor: "pointer",
+    alignItems: "center",
+    backgroundColor: "black",
+    justifyContent: "space-between",
+  },
+  img: {
+    userSelect: "none",
+    height: "100%",
+    pointerEvents: "none"
+  },
+  arrowRight: { 
+    opacity: 0.5,
+    marginRight: "5px", 
+    pointerEvents: "none"
+  },
+  arrowLeft: { 
+    opacity: 0.5,
+    marginLeft: "10px",
```

Comment:
why left and right arrows have different margin? It would be nice to have all margins in a variable in order to be consistent by design.